redskyctl/internal/commands/generate/experiment.go

```diff
@@ -166,9 +169,15 @@ func (o *ExperimentOptions) defaultNamespace() string {
 }
 
 func (o *ExperimentOptions) defaultName() string {
+	// Use the working directory
+	filename := o.Filename
+	if filename == "" || filepath.Dir(filename) == "/dev/fd" {
+		filename = "./default"
```

Comment:
The `default` part is actually ignored. It could have been `./FUZZYBUNNY`, all we are after is the working directory name (e.g. it could have been `./app.yaml`, which illustrates why we are taking the dirname below).